source/nodejs/adaptivecards/src/schema.ts

```diff
@@ -27,17 +27,22 @@ export interface IBackgroundImage {
     url: string;    
 }
 export interface IAdaptiveCard extends ITypedElement {
-    version?: IVersion | string,
-    actions?: (IActionBase)[];
+    type: string;
```

Comment:
ITypedElement already has a type property. Either remove ITypedElement or remove type for IAdapativeCard